.github/workflows/ci.yaml

```diff
@@ -35,7 +35,6 @@ jobs:
     strategy:
       matrix:
         php:
-          - '7.2'
```

Comment:
Dropping PHP version in this PR is IMHO no-go. This should be done in another one, cleaning just removes code in fact dead (Symfony <4.4 & PHP <7.2).

Reply:
I'm sorry, I thought it was already dropped. I'll create another PR for it.